chartpress.py

```diff
@@ -566,20 +567,37 @@ def publish_pages(chart_name, chart_version, chart_repo_github_path, chart_repo_
     this, it is as we would have a --force-publish-chart by default.
     """
 
-    # clone the Helm chart repo and checkout its gh-pages branch,
-    # note the use of cwd (current working directory)
-    checkout_dir = '{}-{}'.format(chart_name, chart_version)
-    check_call(
-        [
-            'git', 'clone', '--no-checkout',
-            git_remote(chart_repo_github_path),
-            checkout_dir,
-        ],
-        # warning: if echoed, this call could reveal the github token
-        echo=True,
-    )
+    # clone/fetch the Helm chart repo and checkout its gh-pages branch, note the
+    # use of cwd (current working directory)
+    checkout_dir = f'{chart_name}-{chart_version}'
+    if not os.path.isdir(checkout_dir):
+        check_call(
+            [
+                'git', 'clone', '--no-checkout',
+                git_remote(chart_repo_github_path),
+                checkout_dir,
+            ],
+            # warning: if echoed, this call could reveal the github token
+            echo=True,
+        )
+    else:
+        check_call(['git', 'fetch'], cwd=checkout_dir, echo=True)
```

Comment:
`chartpress --publish-chart` leads to a directory being created. If you try to publish the same chart version twice with an existing directory you get an error with `git clone`. This avoids that by reusing the previous git clone if it exists already.